.github/workflows/ci.yml

```diff
@@ -85,6 +85,8 @@ jobs:
       uses: pypa/cibuildwheel@v2.11.2
       env:
         CIBW_BUILD: "cp*-manylinux_x86_64 cp3*-win_amd64 cp3*-macosx_x86_64"
+        CIBW_ENVIRONMENT_MACOS: "CFLAGS=-DNDEBUG -g0"
+        CIBW_ENVIRONMENT_LINUX: "CFLAGS=-DNDEBUG -g0 -mssse3"
```

Comment:
This should be configured in `pyproject.toml` (where there already is a `[tool.cibuildwheel]` section)

I moved it to `pyproject.toml` because then running `cibuildwheel` locally and on CI use the same options (`CIBW_BUILD` remains because this needs to be overriden anyway when running locally).